WBooks/Repositories/BooksRepository.swift

```diff
@@ -57,5 +55,8 @@ class BooksRepository: AbstractRepository, BooksRepositoryType {
         let path = Constants.booksList + "/\(id)/comments"
         return performRequest(method: .get, path: path) { decode($0).toResult()}
     }
-    
+
+    func submitBook() -> SignalProducer<Void, RepositoryError> {
+        return SignalProducer(value: ())
```

Comment:
Add a comment saying APi is not working yet or soemthing. Like if it was a true project.